src/main/scala/ai/privado/cache/AuditCache.scala

```diff
@@ -48,6 +47,18 @@ object AuditCache {
     ruleCache: RuleCache
   ): Unit = {
 
+    dataflowMapByPathId = getCalculatePathIdAndStorePath(dataflowPathsUnfiltered)
+
+    val expendedSourceSinkInfo =
+      DuplicateFlowProcessor.processExpendedSourceSinkData(dataflowMapByPathId, privadoScanConfig, ruleCache, false)
+
+    expendedSourceSinkInfo.foreach(flowInfo => {
+      addIntoBeforeFirstFiltering(SourcePathInfo(flowInfo.pathSourceId, flowInfo.sinkId, flowInfo.sinkPathId))
+    })
+  }
+
+  private def getCalculatePathIdAndStorePath(dataflowPathsUnfiltered: List[Path]): Map[String, Path] = {
```

Comment:
Rename the function to `getDataflowPathAndIdMap`

Reply:
okay sure.